src/Common/Storage/Commands.Storage.Test/packages.config

```diff
@@ -2,7 +2,7 @@
 <packages>
   <package id="Hyak.Common" version="1.0.2" targetFramework="net45" />
   <package id="Microsoft.Azure.Common" version="2.1.0" targetFramework="net45" />
-  <package id="Microsoft.Azure.Common.Authentication" version="1.3.3-preview" targetFramework="net45" />
+  <package id="Microsoft.Azure.Common.Authentication" version="1.4.0-preview" targetFramework="net45" />
```

Comment:
This is required as I have changes to the Authentication package that are required by the new services I am adding. I chose to upgrade all of the existing powershell services in order to keep them at the same level. Once my SDK changes are approved, my plan is to downlevel the authentication package back to the current version and send a new PR with those updates for all services.